src/libs/shim-proto/README.md

```diff
@@ -0,0 +1,40 @@
+# shim-proto
+
+Protobuf message and ttRPC service definitions for Containerd shim v2 protocol.
+
+## Design
+
+The `shim-proto` crate provides [Protobuf](https://github.com/protocolbuffers/protobuf.git) message
+and [ttRPC](https://github.com/containerd/ttrpc.git) service definitions for the
+[Containerd shim v2](https://github.com/containerd/containerd/blob/main/runtime/v2/task/shim.proto) protocol.
+Details about `shim-v2` can be found [here](https://github.com/kata-containers/kata-containers/tree/main/docs/design/architecture#shim-v2-architecture)
```

Comment:
This should be a relative URL I think so that it wil work if we renamed the repo, but also will continue to work when it's copied into a stable branch.